### PR TITLE
Fix settings on develop

### PR DESCRIPTION
**Pull request by kamils-iRonin**

### CODE REVIEW

File: app/models/setting/netbox.rb [hunk 1]
```diff
@@ -20,7 +20,10 @@ def self.load_defaults
       return unless super
 
       transaction do
-        default_settings.each { |s| create! s.update(category: 'Setting::Netbox') }
+        default_settings
+          .map { |s| s.merge(category: 'Setting::Netbox') }
+          .map { |s| s.slice(*column_names.map(&:to_sym)) }
+          .map { |s| find_or_initialize_by(s.slice(:name)).update!(s) }
```
Comment (laugmanuel) [inline app/models/setting/netbox.rb:26]: Should this be `.each`?

Comment (kamils-iRonin) [reply app/models/setting/netbox.rb:26]: it doesn't really matter in this case, both iterate over the array

Comment (laugmanuel) [reply app/models/setting/netbox.rb:26]: yeah - I just find it weird when `map` changes external data.

Comment (kamils-iRonin) [reply app/models/setting/netbox.rb:26]: it doesn't really change the external data, but creates a new array containing the values returned by the block

**Pull request merged**